src/sentry/testutils/pytest/relay.py

```diff
@@ -78,9 +76,13 @@ def relay_server_setup(live_server, tmpdir_factory):
     )
     assert redis_db == projectconfig_backend.cluster.connection_pool.connection_kwargs["db"]
 
+    with socket.socket() as sock:
+        sock.bind(("127.0.0.1", 0))
+        RELAY_PORT = sock.getsockname()[1]
+
     template_vars = {
         "SENTRY_HOST": f"http://host.docker.internal:{port}/",
-        "RELAY_PORT": relay_port,
+        "RELAY_PORT": RELAY_PORT,
```

Comment:
there's still a chance the port gets taken in the time between here and when the docker container is created - should i try and see if i can move stuff around to get as close as possible to `docker_client.containers.run(**options)` or is this good enough?